assets/src/blocks/Cookies/CookiesScript.js

```diff
@@ -1,9 +1,8 @@
 import {createRoot} from 'react-dom/client';
-import {BLOCK_NAME} from './CookiesBlock';
 import {CookiesFrontend} from './CookiesFrontend';
 
 // Fallback for non migrated content. Remove after migration.
-document.querySelectorAll(`[data-render="${BLOCK_NAME}"]`).forEach(
+document.querySelectorAll('[data-render="planet4-blocks/cookies"]').forEach(
```

Comment:
This is the actual fix, because importing something from the `CookiesBlock` also imported the `CookiesEditor` component and caused the problem.